### DISCUSSION TITLE
Blank website, with just the readme

### DISCUSSION THREAD

**Discussion by josebbento** [community/community#108092] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

This week I was making a website, so far so good, the problem was when it came to hosting. I always host my website on github and it works, but there (when I access the link that github creates) a blank screen appears, only with what is written in the readme even with all the files, I tried publishing a bunch of times, but always results in the same thing

![Captura de tela de 2024-02-12 16-48-20](https://github.com/community/community/assets/81451729/30b8bb09-e7eb-4d8b-a15e-f766d99b856a)

Links
[page](https://josebbento.github.io/Celular/)
[directory](https://github.com/josebbento/Celular)

resolution:

there was no "index.html" file

**Comment by ppande574** [comment]
I'm having the same issue with my website [here](https://github.com/ppande574/Personal_Website). It's my first time trying to publish a website through Github pages, but the exact same thing is happening to me as well, where all that comes up is my "Readme.md" file instead of the actual website that's in the other folder. 

I'm deploying from "main" and it doesn't let me select any other folders to deploy other than "root" or "docs" either.

**Reply by josebbento** [reply]
is that you need an "index.html" file in the root of the repository. that's what worked
